improver/wxcode/weather_symbols.py

```diff
@@ -256,10 +260,7 @@ def invert_condition(test_conditions):
 
         return inverted_threshold, inverted_combination
 
-    @staticmethod
-    def construct_condition(
-        extract_constraint, condition, probability_threshold, gamma
-    ):
+    def construct_condition(self, extract_constraint, condition, probability_threshold):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/manual/metoppv_tech/improver/pullRequest?prid=5334158)